projects/ngx-formentry/src/components/radio-button/radio.component.css

```
@@ -0,0 +1,4 @@
+.in-line {
+  display: inline-block;
+  padding-right: 0.5rem;
+}
```

```suggestion
.inline {
  display: inline-block;
  padding-right: 0.5rem;
}
```
I've tried this combination of styles on my setup and they don't seem to achieve the desired result.  I suspect what ought to happen is that the following styles should be applied to the parent `fieldset` tag:

```css
.inline {
  display: flex;
  gap: 0.5rem; // optional
  flex-direction: row;
}
```